assets/src/dashboard/components/navigation-bar.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { useRouteHistory } from '../app/router';
+import WebStoriesLogo from '../images/logo.svg';
```

Comment:
Per the other thread, let's do the same as [here](https://github.com/google/web-stories-wp/blob/192d430bb7ffc2dd17fedf191136736640da8770/assets/src/edit-story/components/button/index.js#L26). I think the tool we already use is svgo.